gdk/wizard/WizardChecker.py

```diff
@@ -0,0 +1,78 @@
+from gdk.wizard.ConfigEnum import ConfigEnum
+import re
+import ast
+
+
+class WizardChecker:
+    def is_valid_input(self, input_value, field):
+        """
+        Prompts the user of all the optional ConfigEnum of the gdk config file and updates the
+        field_map if their answer is valid  as the user answers the question to each prompt
+
+        Parameters
+        ----------
+            input_value(string): The value entered by the user for the field
+            field(string): The field name of the field in the gdk config file
+
+        Returns
+        -------
+            boolean: True if the input value is valid for the field, False otherwise.
+        """
+
+        switch = {
+            ConfigEnum.AUTHOR: self.check_author,
+            ConfigEnum.VERSION: self.check_version,
+            ConfigEnum.CUSTOM_BUILD_COMMAND: self.check_custom_build_command,
+            ConfigEnum.BUILD_SYSTEM: self.check_build_system,
+        }
+        return switch.get(field)(input_value)
+
+    def check_author(self, input_value):
+        # input must be a non-empty string
+        return isinstance(input_value, str) and len(input_value) > 0
```

Comment:
What about max length? This limit should be same as what Greengrass supports.

Reply:
I just based my checks of what the schema specified. I looked at here https://docs.aws.amazon.com/greengrass/v2/developerguide/gdk-cli-configuration-file.html#gdk-config-format
to see the speifications for each field was but I didnt see anything about the max length